federatedscope/attack/example_attack_config/CRA_fedavg_convnet2_on_femnist.yaml

```diff
@@ -16,6 +16,7 @@ data:
   batch_size: 10
   subsample: 0.0001
   num_workers: 0
+  transform: 'transforms.ToTensor(), transforms.Normalize(mean=[0.1307], std=[0.3081])'
```

Comment:
@xieyxclack @yxdyc @DavdGao  This kind of args is very tricky. Do you have any better solution?

Reply:
Maybe we can add a generator for transforms to support different backends, such as:
```
def get_transform(trans):
     if cfg.backend == 'torch':
        ...
    elif cfg.backend == 'tf':
        ...

```
`trans` is the selected transforms, e.g. `trans=["resize", "crop", "normalize", "flip"]`, that can be translated into the corresponding operations in torch/tensorflow.

Reply:
Oh...IMP, this kind of args should not be allowed (like injection?). 
Maybe we can pre-define some widely-used operations (such as `ToTensor`, `Normalize`, 'Resize') and the config only contains the names of operations. But I am not sure such `transform` can only be used for torchvision datasets?

Reply:
> Maybe we can add a generator for transforms to support different backends, such as:
> 
> ```
> def get_transform(trans):
>      if cfg.backend == 'torch':
>         ...
>     elif cfg.backend == 'tf':
>         ...
> ```
> 
> `trans` is the selected transforms, e.g. `trans=["resize", "crop", "normalize", "flip"]`, that can be translated into the corresponding operations in torch/tensorflow.

This has been done, see `get_transform` from `transform_builder`. We should a better way to handle the args of these transforms.

Reply:
As for more complex transforms, e.g. `Normalize`, that require additional parameters (mean, std), we may preset  them in `get_data` for the common datasets like cifar10/100, femnist or celebA. 
Also, we should support customized mean/std in certain way, like`trans=["normalize:{'mean':0., 'std':1.}"]`